google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1153,6 +1159,19 @@ public Builder setAsyncExecutorProvider(CloseableExecutorProvider provider) {
       return this;
     }
 
+    /**
+     * Sets the {@link DirectedReadOption} that specify which replicas or regions should be used for
+     * non-transactional reads or queries.
+     *
+     * <p>DirectedReadOptions set at the request level will take precedence over the options set
+     * using this method.
+     */
+    public Builder setDirectedReadOption(DirectedReadOptions directedReadOptions) {
```

Comment:
Should we add a concrete example to this Javadoc for how to create a `DirectedReadOptions` instance? I do not feel that the `DirectedReadOptions` class and javadoc is self-explanatory.

Reply:
I have added one example of how the object is built. Our samples will also contain more usage patterns for each language. LMK if something else needs to be included.

Reply:
Thanks, this looks fine to me.